catalogue/aarch64-ifetch/tests/MP+rel+blr.litmus

```diff
@@ -0,0 +1,14 @@
+AArch64 MP+rel+blr
```

Comment:
For this test to run on hardware, one must declare `X30` to be allocated  to itself by gcc:
```suggestion
AArch64 MP+rel+blr
Stable=X30
```
See also https://github.com/herd/herdtools7/pull/847#discussion_r1622564008, as printed `1:X6` is of instruction ttype.